R/understat_match_players.R

```diff
@@ -0,0 +1,74 @@
+
+#' Get Understat match player data
+#'
+#' Returns player values for a selected match from Understat.com.
+#'
+#' @param match_url A `character` string with the URL of the match played.
+#'
+#' @return returns a `data.frame` with data for all players for the match.
+#'
+#' @importFrom magrittr %>%
+#' @importFrom rvest html_elements
+#' @importFrom stringi stri_unescape_unicode
+#' @importFrom stringr str_subset
+#' @importFrom jsonlite fromJSON
+#' @importFrom dplyr bind_rows
+#'
+#' @export
+
+understat_match_players <- function(match_url) {
```

Comment:
my only new feedback on this is that there might have been a cleaner way of doing all of this. i noticed that [`{understatr}` seems to have a more straightforward approach](https://github.com/ewenme/understatr/blob/dd6e48fb3b86174b9346e9b08a821045c9415d4d/R/matches.R), although i haven't thoroughly checked the details. it could be the same exact implementation.

Reply:
but i don't think this function should be changed at this point